### PR TITLE
Libhandy upstream sync

### PR DESCRIPTION
**Pull request by Jupiter007-43**
- Create sync script
- First sync all files from upstream
- Add Github action

@clobrano it is ok?

### CODE REVIEW

File: gtk/upstream/libhandy/libhandy-sync.sh [hunk 1]
```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
```
Comment (clobrano) [inline gtk/upstream/libhandy/libhandy-sync.sh:1]: I think we don't need this file.

File: .github/workflows/open-pr-on-changes.yaml [hunk 2]
```diff
@@ -16,6 +16,7 @@ jobs:
           { project: GNOME Shell, repo: "https://gitlab.gnome.org/GNOME/gnome-shell.git", branch: master, projectdir: gnome-shell, src: data/theme/, dest: gnome-shell/upstream },
           { project: GNOME Shell Resources, repo: "https://gitlab.gnome.org/GNOME/gnome-shell.git", branch: master, projectdir: gnome-shell, src: data/gnome-shell-theme.gresource.xml, dest: gnome-shell/upstream/data/gnome-shell-theme.gresource.xml },
           { project: GTK, repo: "https://gitlab.gnome.org/GNOME/gtk.git", branch: gtk-3-24, projectdir: gtk, src: gtk/theme/Adwaita/, dest: gtk/upstream/gtk+3.0/Adwaita },
+          { project: Libhandy, repo: "https://gitlab.gnome.org/GNOME/libhandy.git", branch: master, projectdir: gtk, src: src/themes/, dest: gtk/upstream/libhandy/Adwaita },
```
Comment (clobrano) [inline .github/workflows/open-pr-on-changes.yaml:19]: This is the name of the cloned project, so it is probably "libhandy"

Comment (clobrano) [reply .github/workflows/open-pr-on-changes.yaml:19]: For symmetry, I would keep the same name of the dest dir "gtk/upstream/libhandy/themes"

**Review by clobrano — CHANGES_REQUESTED** [review]

File: .github/workflows/open-pr-on-changes.yaml [hunk 3]
```diff
@@ -16,7 +16,7 @@ jobs:
           { project: GNOME Shell, repo: "https://gitlab.gnome.org/GNOME/gnome-shell.git", branch: master, projectdir: gnome-shell, src: data/theme/, dest: gnome-shell/upstream },
           { project: GNOME Shell Resources, repo: "https://gitlab.gnome.org/GNOME/gnome-shell.git", branch: master, projectdir: gnome-shell, src: data/gnome-shell-theme.gresource.xml, dest: gnome-shell/upstream/data/gnome-shell-theme.gresource.xml },
           { project: GTK, repo: "https://gitlab.gnome.org/GNOME/gtk.git", branch: gtk-3-24, projectdir: gtk, src: gtk/theme/Adwaita/, dest: gtk/upstream/gtk+3.0/Adwaita },
-          { project: Libhandy, repo: "https://gitlab.gnome.org/GNOME/libhandy.git", branch: master, projectdir: gtk, src: src/themes/, dest: gtk/upstream/libhandy/Adwaita },
+          { project: libhandy, repo: "https://gitlab.gnome.org/GNOME/libhandy.git", branch: master, projectdir: gtk, src: src/themes/, dest: gtk/upstream/libhandy/themes },
```
Comment (clobrano) [inline .github/workflows/open-pr-on-changes.yaml:19]: Sorry, I did't write what field you should change, my bad.
The "project" field was fine (Libhandy), the field to change was "projectdir", from "gtk" to "libhandy".

Comment (Jupiter007-43) [reply .github/workflows/open-pr-on-changes.yaml:19]: No, problem!

**Review by clobrano — APPROVED** [review]
It looks good now. There are CI errors, but one seems due to the GitHub action we use for PR creation, the other is unlikely due to this change.

**Comment by Jupiter007-43** [comment]
Great, thanks for the review 👍
Can I merge?

**Comment by clobrano** [comment]
Sure :)

**Pull request merged**